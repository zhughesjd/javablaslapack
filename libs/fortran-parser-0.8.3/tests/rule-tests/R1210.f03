! Test external-stmt
!      external-stmt  is  EXTERNAL [ :: ] external-name-list
!
! Not tested here: external-name-list
external a
external a, b
external :: a
external :: a, b

end

