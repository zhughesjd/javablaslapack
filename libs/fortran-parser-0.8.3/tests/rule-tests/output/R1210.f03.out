*******************************************
args[1]: R1210.f03
R1101:main-program__begin:
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@10,131:131='a',<209>,5:9]
R 102:generic-name-list: count=1
R1210:external-stmt:
R 212:specification-stmt:
R 207:declaration-construct:
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@14,142:142='a',<209>,6:9]
R 102:generic-name-list-part: id=[@17,145:145='b',<209>,6:12]
R 102:generic-name-list: count=2
R1210:external-stmt:
R 212:specification-stmt:
R 207:declaration-construct:
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@23,159:159='a',<209>,7:12]
R 102:generic-name-list: count=1
R1210:external-stmt:
R 212:specification-stmt:
R 207:declaration-construct:
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@29,173:173='a',<209>,8:12]
R 102:generic-name-list-part: id=[@32,176:176='b',<209>,8:15]
R 102:generic-name-list: count=2
R1210:external-stmt:
R 212:specification-stmt:
R 207:declaration-construct:
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=4
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
Parser exiting normally
********************************************
